DopplerFiles.Test/DopplerFiles.Test.csproj

```diff
@@ -1,16 +1,31 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
-
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
What is the difference between `net5.0` and `netcoreapp5.0`?

Reply:
There are no differences. I've forgotten revert it when I updated the Test project